pkg/api/conversion.go

```diff
@@ -248,3 +252,16 @@ func Convert_unversioned_LabelSelector_to_map(in *metav1.LabelSelector, out *map
 	}
 	return err
 }
+
+func Convert_Slice_string_To_Slice_int32(in *[]string, out *[]int32, s conversion.Scope) error {
```

Comment:
@kargakis How would you like it documented? The intent, a reference to where it is used? We have a reverse issue that you get this behavior for free once you add it so others could rely on the conversion without realizing it.